### PR TITLE
Added also UV maximum to history page

### PR DESCRIPTION
**Pull request by seehase**

### CODE REVIEW

**Comment by seehausen** [comment]
any updates on my pull request?

**Comment by ngulden** [comment]
I'm sorry, but I didn't have the time yet to take a look.

File: skins/niculskin/index.html.tmpl [hunk 1]
```diff
@@ -78,8 +78,22 @@
                         <tr><td>$niculskinLabels.conditions.wind</td><td>$current.windSpeed $current.windDir.ordinal_compass ($current.windDir)</td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_rate</td><td>$current.rainRate</td></tr>
                         <tr><td>$niculskinLabels.conditions.in_temp_humidity</td><td>$current.inTemp | $current.inHumidity</td></tr>
+                        #if $day.extraTemp1.has_data
                         <tr><td>$niculskinLabels.conditions.extra1_temp_humidity</td><td>$current.extraTemp1 | $current.extraHumid1</td></tr>
+                        #end if
+                        #if $day.extraTemp1.has_data
```
Comment (ngulden) [inline skins/niculskin/index.html.tmpl:84]: Please see https://github.com/ngulden/niculskin/pull/1#pullrequestreview-94105600

**Review by ngulden — CHANGES_REQUESTED** [review]
This looks like the same patch as in https://github.com/ngulden/niculskin/pull/1. Please make the changes asked for in that pull request and submit them. Afterwards, you can rebase this pull request on those changes and I'd happily accept them.

**Comment by seehase** [comment]
Changed everything to use lables and fixed all of your comments

**Review by ngulden — APPROVED** [review]
Thanks for the adjustments. They look good.

**Comment by ngulden** [comment]
@seehase : Thank you very much for you contribution.

**Pull request merged**